src/Pages/Basket/Basket.js

```diff
@@ -0,0 +1,116 @@
+import React, { Component } from 'react';
+import MainBasket from './MainBasket/MainBasket';
+import '../../styles/reset.scss';
+import './Basket.scss';
+
+class Basket extends Component {
+  constructor() {
+    super();
+    this.state = {
+      orderList: [],
+    };
+  }
+  // [ProductList] map하기
+  componentDidMount() {
+    fetch('/Basket/basket.json')
+      .then(res => res.json())
+      .then(data => {
+        this.setState({ orderList: data });
+      });
+  }
+  //[장바구니] (서버 -> 장바구니) Item 가져오기
+  // [장바구니](장바구니 -> 서버) 장바구니 담은 상품 전달하기
+  giveItemList = () => {
+    // console.log('click');
+    let newOrderList = this.state.orderList.map(order => {
+      return {
+        id: order.id,
+        count: order.count,
+      };
+    });
+    console.log(newOrderList);
+    // fetch('http://localhost:8000/carts'), {
+    fetch('http://10.5.30.109:8000/products/basket', {
+      method: 'POST',
+      body: {
+        orderList: newOrderList,
+      },
+    })
+      .then(res => res.json())
+      .then(data => {
+        // { MESSAGE: SUCCESS }
+        console.log(data);
+      });
+  };
+
+  deleteBasketItem = cool => {
+    // cool 이란 어떤 객체인지 구분해 주는 특정 데이터이다. ex) id
+    let { orderList } = this.state;
+    this.setState({
+      orderList: orderList.filter(order => {
+        console.log('order');
+        console.log(order);
+
+        if (order.id === cool) {
+          return;
+        }
+        return order;
+      }),
+    });
+  };
+
+  changeCount = (id, count) => {
+    if (count === 0) return;
```

Comment:
👍 👍 👍 
```suggestion
  changeCount = (id, count) => {
    if (!count) return;
```
이것도 같은 의미 입니다!